<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="18" column="0" colspan="6">
       <widget class="QPushButton" name="pbResume">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Resume</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="6">
       <widget class="QPushButton" name="pbNewProject">
        <property name="text">
         <string>New Project...</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="6">
       <widget class="QPushButton" name="pbQuit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lTimeSpan">
        <property name="text">
         <string>Time Span</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="6">
       <widget class="QPushButton" name="pbGraph">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Create Citation Graph</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lFolder">
        <property name="text">
         <string>Work Folder</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0" colspan="6">
       <widget class="QPushButton" name="pbPause">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="6">
       <widget class="QPushButton" name="pbSearch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lTopic">
        <property name="text">
         <string>Search Query</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="leTimeSpanFrom">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QPushButton" name="pbLoadProject">
        <property name="text">
         <string>Load Project...</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="6">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="pbQuery">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QLineEdit" name="leQuery">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="lTimeSpanFrom">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QLineEdit" name="leFolder">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLineEdit" name="leProjectName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="6">
       <widget class="QPushButton" name="pbSearchCitation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Get Citations</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="lTimeSpanTo">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QLineEdit" name="leTimeSpanTo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="6">
       <widget class="QPushButton" name="pbCreateDB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Create Database</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="6">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lProjectName">
        <property name="text">
         <string>Project Name</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1" colspan="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Task Progress</string>
        </property>
       </widget>
      </item>
      <item row="15" column="5">
       <widget class="QLabel" name="lProgress">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>10000/10000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QPushButton" name="pbSaveProject">
        <property name="text">
         <string>Save Project</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionNew_Project</sender>
   <signal>triggered()</signal>
   <receiver>pbNewProject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_Project</sender>
   <signal>triggered()</signal>
   <receiver>pbLoadProject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
